src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,47 @@
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    display: flex;
+    margin-right: 20px;
+    text-transform: uppercase;
+
+    @extend %typography;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    padding: 22.5px 0;
```

Comment:
```suggestion
    padding: 23px 0;
```
![image](https://user-images.githubusercontent.com/92453897/186729412-593314eb-545e-4754-9076-89812f2db309.png)

Reply:
adjusted to 23px